src/main/java/com/github/manolo8/darkbot/core/manager/SettingsManager.java

```diff
@@ -36,6 +36,8 @@ public void tick() {
         this.config = API.readMemoryInt(address + 92);
 
         this.nextMap = API.readMemoryInt(address + 244);
+        this.main.mapManager.checkNextMap(nextMap);
+
```

Comment:
```suggestion
```
Instead of settings manager having to reach into map manager, make map manager check this instead